site/content/docs/browser-checks/scraping-onpage-elements.md

```diff
@@ -1,154 +1,142 @@
 ---
 title: Scraping & asserting on page elements
-description: How to scrape web page elements like texts, buttons and forms with Google Chrome and Puppeteer or Playwright.
+description: How to scrape web page elements like texts, buttons and forms with Google Chrome and Playwright Test.
 weight: 15
 menu:
   docs:
     parent: "Browser checks"
 ---
 
 Any standard Node.js script that successfully finishes an execution is a valid, passing browser check. However, in
-many cases you want to validate whether the session is actually showing the right stuff to the user.
+many cases, you want to validate whether the session shows the right stuff to the user.
 
 This is a two-step process:
 
 1. Scrape the relevant item from the current page.
 2. Assert that it shows what you expect.
 
-Both Playwright and Puppeteer offer many ways to scrape elements like buttons, forms or any arbitrary HTML element. We've listed the most
-used ones below, together with some tips on how to use them effectively.
+Playwright Test offers many ways to scrape elements like buttons, forms or any arbitrary HTML element. We've listed the most
+common ones below, together with some tips on how to use them effectively.
 
 ## Scraping text values
 
-Both libraries provide the `page.$eval()` method which is a very powerful way to actually run a selector query inside a web page. See the example
-below. Note: we are using Jest's `expect` library for assertions here.
+Playwright Test uses Jest's [`expect`](https://jestjs.io/docs/expect) library for test assertions. Playwright also extends it with convenience async matchers that will wait until the expected condition is met.
```

Comment:
This intro to expect should go to the intro section of the page. Please add a mention of web-first assertions and a link to PWT docs https://playwright.dev/docs/test-assertions#list-of-assertions